pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -449,12 +449,16 @@ func (m *cgroupManagerImpl) Pids(name CgroupName) []int {
 
 		// WalkFunc which is called for each file and directory in the pod cgroup dir
 		visitor := func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				glog.V(5).Infof("cgroup manager encountered error scanning cgroup path %q", path)
+				return filepath.SkipDir
```

Comment:
The problem is that if there is an error, info may be nil. And this will cause kubelet panic.